packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexPageHeader.tsx

```diff
@@ -17,10 +25,23 @@ export const RecordIndexPageHeader = () => {
   const { findObjectMetadataItemByNamePlural } =
     useFilteredObjectMetadataItems();
 
-  const { objectNamePlural, onCreateRecord } = useContext(
+  const { objectNamePlural, onCreateRecord, recordIndexId } = useContext(
     RecordIndexRootPropsContext,
   );
+  const { records: views } = usePrefetchedData<View>(PrefetchKey.AllViews);
```

Comment:
this should be loaded in the Dropdown, not in the Header, this will trigger un ncessary re-renders